docs/_includes/masthead.html

```diff
@@ -30,9 +30,8 @@
                             <div class="id7-search-box-container floating-label">
                                 <input type="search" class="form-control input-lg" id="id7-search-box" name="q" placeholder="Search Warwick" data-suggest="go" aria-label="Search Warwick">
                                 <label for="id7-search-box">Search Warwick</label>
-                                <i class="fas fa-search fa-2x" role="button" aria-label="Perform Search" tabindex="0"></i>
+                                <button type="submit" class="search-button"><span class="sr-only">Perform search</span><i class="fas fa-search fa-2x" title="Magnifying glass"></i></button>
```

Comment:
Not sure we need the title on the icon and I'd just make the button label "Search", as you'd expect a normal Search button to be labelled, rather than "Perform search"?